code/modules/modular_computers/file_system/programs/warrant.dm

```diff
@@ -0,0 +1,33 @@
+/datum/computer_file/program/warrant
+	filename = "warrantapp"
+	filedesc = "Citations & Fines"
+	download_access = list(ACCESS_BRIG_ENTRANCE)
+	downloader_category = PROGRAM_CATEGORY_SECURITY
+	program_open_overlay = "warrant"
+	extended_desc = "This program allows any user to read, pay, and print citations on go."
+	size = 2
+	program_flags = PROGRAM_ON_NTNET_STORE | PROGRAM_REQUIRES_NTNET
+	can_run_on_flags = PROGRAM_PDA
+	tgui_id = "NtosWarrant"
+	program_icon = "money-check-dollar"
+	///The UI we use for the citations
+	var/obj/machinery/computer/warrant/warrant_ui
```

Comment:
This bit seems a tad too hacky: I suggest you make a datum that manages the ui data and actions in the similar style of `GLOB.crewmonitor`, or at the very least, make this a static variable unless there're issues with having multiple concurrent users, even if the latter option will leave me a little bitter.